src/protocol/mod.rs

```diff
@@ -50,6 +50,13 @@ pub struct WebSocketConfig {
     /// be reasonably big for all normal use-cases but small enough to prevent memory eating
     /// by a malicious user.
     pub max_frame_size: Option<usize>,
+    /// If set to true it will allow the websocket server to accept unmasked frames from client.
+    /// Even though this behaviour is not in compliance with RFC 6455 (which requires the server
+    /// to close the connection when unmasked frame from client is received) it might be handy in some cases
+    /// as there are existing applications sending unmasked client frames. By default (i.e. unless not explicitly specified)
+    /// this flag is set to false which means violation of RFC 6455 is not allowed and unmasked client frames will be rejected
+    ///by websocket server.
+    pub server_allow_unmasked: bool,
```

Comment:
A couple minor remarks:

1. I would call it `accept_unmasked_frames` (a bit more descriptive name), what do you think?
2. There is a space missing after the `///` at the last line.
3. I would change the description a bit, text below:

```
/// When set to `true`, the server will accept and handle unmasked frames
/// from the client. According to the RFC 6455, the server must close the
/// connection to the client in such cases, however it seems like there are
/// some popular libraries that are sending unmasked frames, ignoring the RFC.
/// By default this option is set to `false`, i.e. according to RFC 6455.
```